spec/resource_spec.rb

```diff
@@ -73,35 +98,26 @@
   end
 
   describe '#default_sort' do
-    it 'gets/sets correctly' do
-      klass.default_sort([{ name: :desc }])
-      expect(instance.default_sort).to eq([{ name: :desc }])
-    end
-
     it 'defaults' do
       expect(instance.default_sort).to eq([])
     end
   end
 
   describe '#default_page_size' do
-    it 'gets/sets correctly' do
-      klass.default_page_size(10)
-      expect(instance.default_page_size).to eq(10)
-    end
-
     it 'defaults' do
       expect(instance.default_page_size).to eq(20)
     end
   end
 
   describe '#type' do
-    it 'gets/sets correctly' do
-      klass.type :authors
-      expect(instance.type).to eq(:authors)
+    it 'defaults' do
+      expect(instance.type).to eq(:undefined_jsonapi_type)
```

Comment:
Any interest in trying to infer a type based on class name?

Reply:
Probably - I'm working on attributes w/resource serializer now, and we'll have to infer a serializer name, so I'll see if it makes sense then.

Reply:
@wadetandy fyi - done now